gettsim/parameters/ges_rente.yaml

```diff
@@ -759,6 +771,184 @@ regelaltersgrenze:
+    en:
+      Women born before 1940 were able to retire at age 60 without deductions (required waiting period of 15 years and 10 years of
+      contributions after age 40). The age threshold was increase by one month for each birthmonth up to age 65 for cohort 1945.
+      Between cohort 1945 and 1951 the threshold remained constant.
+      The special pension scheme for women was abolished for cohorts born afer 1952. regelaltersgrenze afterwards.
+  type: piecewise_linear
+  1980-01-01:
+    0:
+      lower_threshold: -inf
+      upper_threshold: 1939.916667
+      rate_linear: 0
+      intercept_at_lower_threshold: 60
+    1:
+      upper_threshold: 1944.916666666
+      rate_linear: 1
+    2:
+      upper_threshold: 1951.916666666 # constant 65
+      rate_linear: 0
+    3:
+      upper_threshold: 1952.0000 # since 1952 abolished -  regelaltersgrenze applies - jump in theshold by 6 month
+      rate_linear: 6
+    4:
+      upper_threshold: 1958
+      rate_linear: 0.083333333
+    5:
+      upper_threshold: 1964
+      rate_linear: 0.16666666
+    6:
+      upper_threshold: inf
+      rate_linear: 0
+
+altersgrenze_für_frauen_vorzeitig:
+  name:
+    de:
+    en:
+  description:
+    de:
+    en:
+  reference:
+  type: piecewise_linear
+  1980-01-01:
+    0:
+      lower_threshold: -inf
+      upper_threshold: 1952
+      rate_linear: 0
+      intercept_at_lower_threshold: 60
+    1:
+      upper_threshold: 1953
+      rate_linear: 7
+    2:
+      upper_threshold: inf # constant 67
+      rate_linear: 0
+
+altersgrenze_langjährig_versicherte_vorzeitig:
+  name:
+    de:
+    en:
+  description:
+    de:
+    en:
+  reference:
+  1980-01-01:
+    scalar: 63
+
+altersgrenze_langjährig_versicherte_abschlagsfrei:
+  name:
+    de: Eintrittsalter für langjährig Versicherte - abschlagsfrei
+    en: Entry age for long term insured (35 years) without deductions
+  description:
+    de:
+      Stückweise lineare Funktion, die die Regelaltersgrenze angibt, bei der das Individuum mit seinem vollen Rentenanspruch in Rente
+      gehen kann. Geht die Person früher oder später in Rente, ist der Zugangsfaktor und damit der Rentenanspruch höher oder niedriger,
+      wenn keine Ausnahmeregelungen erfüllt sind.
+    en:
+      Piecewise linear function returning the regelaltersgrenze at which the agent is allowed to get pensions with his full
+      claim. If the agent retires earlier or later, the Zugangsfaktor and therefore
+      the pension claim is higher or lower, unless special conditions apply.
+  reference: § 236 SGB VI,
+    https://sozialversicherung-kompetent.de/rentenversicherung/leistungsrecht/242-altersrente-fuer-langjaehrig-versicherte-altersgrenzen.html
+  note:
+    de: >-
+      Kohorten vor 1937 konnten mit 63 abschlagsfrei in Rente gehen.
+      Ab Geburtsdatum Januar 1937 wurde fuer jeden Monat, den die Person spaeter geboren wurde das
```

Comment:
Can you search and replace to use proper Umlaute and ß, please?

Reply:
ok